beherabikram

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "Sara Jaoude",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"
```

Comment:
There is a comma (**,**) missing on line 3, rendering the json file invalid.
```suggestion
  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
```

Reply:
I will mark this conversation as resolved as this bio should be removed as it is the bio used in the example docs (and my personal bio).